contrib/rpm/buildah.spec

```diff
@@ -91,6 +91,99 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
+* Sat Jun 9 2018 Dan Walsh <dwalsh@redhat.com> 1.1-1
```

Comment:
This line here needs 1.2-dev to pass the check as that is what buildah is outputting, really should be 1.2-dev-1... but anyway.